web/source/keymanweb.js

```diff
@@ -3453,6 +3569,42 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
       return ((lg == 'cmn') || (lg == 'jpn') || (lg == 'kor'));
     }
     keymanweb.isCJK = keymanweb['isCJK']; // I3363 (Build 301)
+
+    /**
+     * Function     isChiral
+     * Scope        Public
+     * @param       {Object=}   k0
+     * @return      {boolean}
+     * Description  Tests if the active keyboard (or optional argument) uses chiral modifiers.
+     */
+    keymanweb.isChiral = keymanweb['isChiral'] = function(k0) {
+      return !!(keymanweb.getKeyboardModifierBitmask(k0) & keymanweb['osk'].modifierBitmasks['IS_CHIRAL']);
```

Comment:
What does the `!!` operator do?

Reply:
It's converting the integer result (a bitwise logic operation's result) into a boolean.

```
var a = 5;  // a = 5;
a = !a;     // a is not null or equal to zero, now equals false
return !a;  // invert the prior result - since the original value (of 5) was non-zero and held value, we return true.
```